staging/kubefed/values.yaml

```diff
@@ -33,10 +33,19 @@ controllermanager:
     adoptResources:
   ## Value of feature gates item should be either `Enabled` or `Disabled`
   featureGates:
-    PushReconciler:
-    SchedulerPreferences:
-    CrossClusterServiceDiscovery:
-    FederatedIngress:
+    featureGates:
+      CrossClusterServiceDiscovery: "Disabled"
+      FederatedIngress: "Enabled"
+      PushReconciler: "Enabled"
+      SchedulerPreferences: "Disabled"
+
+  controllermanager:
```

Comment:
Weird nesting - `controllermanager.controllermanager`... If we're going to have separate annotations for webhook and controllermanager, then either we need to add a webhook subchart or we need to pull it all up to the top level chart otherwise this is just weird looking config.

Reply:
> Weird nesting - `controllermanager.controllermanager`... If we're going to have separate annotations for webhook and controllermanager, then either we need to add a webhook subchart or we need to pull it all up to the top level chart otherwise this is just weird looking config.

```
controllermanager <- subchart
  controllermanager <- the deployment
```
this isnt weird, its just unfortunate naming and probably abuse of subcharting tbh.

Reply:
It is odd... if it's the deployment then it should be `controllermanager.deployment`.